tasks/ExtractVersionFromTagTask/extract-version-from-tag.js

```diff
@@ -52,8 +53,17 @@ function run() {
                 originalTag = originalTag.split('\n')[0];
             }
             var tag = originalTag.toLowerCase();
-            if (tag.startsWith(splitPrefix)) {
-                var tagSplitted = tag.split(splitPrefix);
+            var prefixCandidates = Array.from({ length: tag.length }, (_, index) => tag.substring(0, index + 1));
```

Comment:
This is a part I'm not sure about.
There should be a way of extracting the prefix part using the matcher, better than trying matching the prefix by gradually adding more letters and matching each candidates until it doesn't matches anymore...